indico/modules/events/models/events.py

```diff
@@ -419,6 +419,7 @@ def __table_args__(cls):
     end_dt_override = _EventSettingProperty(event_core_settings, 'end_dt_override')
     organizer_info = _EventSettingProperty(event_core_settings, 'organizer_info')
     additional_info = _EventSettingProperty(event_core_settings, 'additional_info')
+    contrib_duration = _EventSettingProperty(event_core_settings, 'contrib_duration')
```

Comment:
I don't think we need this, in the few places where we access it we can get it from the proxy